src/main/java/aquality/selenium/elements/interfaces/IMultiChoiceComboBox.java

```diff
@@ -0,0 +1,44 @@
+package aquality.selenium.elements.interfaces;
+
+public interface IMultiChoiceComboBox extends IComboBox {
+
+    /**
+     * Deselect all selected options
+     */
+    void deselectAll();
```

Comment:
could you add some more functions to this interface please?
As I can see, at least the following could be added:
- [ ] `void selectAll()`
- [ ] `void selectAllByContainingText(String text)`
- [ ] `void selectAllByContainingValue(String value)`
- [ ] `List<String> getSelectedTexts()`
- [ ] `List<String> getSelectedValues()`